docs/copiloting-service.md

```diff
@@ -0,0 +1,119 @@
+
+## What are the aims of the OpenSAFELY copiloting service?
+
+By the end of the 4 week period, pilots should:
+
+* Be familiar with the core OpenSAFELY concepts (study definition, actions, running locally vs running in job server, requesting output release, codelists)
+* Be able to run the OpenSAFELY environment locally and on the job server
+* Have generated the study population for their project
+* Run analysis scripts on the population
+* Generated outputs ready for review
+
+## How does the copiloting service run?
+
+The diagram below illustrates how a project will progress through the copiloting service. More information about each stage is provided in subsections below.
+
+```mermaid
+graph TD
+    subgraph setup_stage[" "]
+        application_submission[Project application]
+        application_approval[Project approval]
+        application_submission --> application_approval
+        getting_started[Completing Getting Started guide]
+    end
+    subgraph assignment_stage[" "]
+            copilot_assignment[Assignment of copilot]
+            introductory_meeting[Introductory meeting]
+            copilot_assignment --> introductory_meeting
+    end
+    subgraph copiloting_stage[" "]
+        subgraph dedicated_support[1:1 support]
+            active_copiloting[Weekly meetings\nCohort review]
+        end
+        subgraph wider_support[Community support]
+            wider_support_details[Discussion forum\nUsers Slack channel]
+        end
+        output_checking[Output checking]
+    end
+    subgraph post_copiloting_stage[" "]
+        subgraph wider_support_post[Community support]
+            wider_support_details_post[Discussion forum\nUsers Slack channel]
+        end
+        manuscript_review[Manuscript review]
+        output_checking_post[Output checking]
+    end
+    getting_started --> assignment_stage
+    application_approval --> assignment_stage
+    introductory_meeting --> copiloting_stage
+    copiloting_stage --> post_copiloting_stage
+    style setup_stage fill:#f7fcb9,stroke:#AAA,stroke-width:4px
+    style assignment_stage fill:#addd8e,stroke:#333,stroke-width:4px
+    style copiloting_stage fill:#31a354,stroke:#333,stroke-width:4px
+    style post_copiloting_stage fill:#fff,stroke:#333,stroke-width:4px
+```
+
+### Project application and software setup
+
+All projects need to have sought and been granted [IG approval](https://www.opensafely.org/policies-for-researchers/) before a copilot can be requested and assigned. We also ask that new users complete the OpenSAFEELY [Getting Started guide](https://docs.opensafely.org/en/latest/getting-started/) to ensure that the necessary software can be installed and run on your computer.
+
+In our experience, copiloting is most successful when the pilot has some experience with coding (e.g., Python, R or Stata), is familiar with collaborative code development via GitHub and is able to commit four weeks or more to their project.
```

Comment:
Is it worth saying that we have had people go through with no experice? This is not the exact phrasing as writing quickly but something like

_However we have had people with motivation, enthusiasm and adjacent skillset get results within four weeks_

Reply:
👍🏻  will add